config/dependency-injection/custom-loader.php

```diff
@@ -0,0 +1,214 @@
+<?php
+/**
+ * Yoast SEO Plugin File.
+ *
+ * @package Yoast\YoastSEO\Dependency_Injection
+ */
+
+namespace Yoast\WP\Free\Dependency_Injection;
+
+use Symfony\Component\Config\FileLocator;
```

Comment:
Is this used in production code, then `YoastSEO_Vendor` should be prefixed for the Symfony use statements.

Reply:
Nope, this only runs in development or when building the artifact.